plugins/pip/pip.plugin.zsh

```diff
@@ -9,6 +9,11 @@
 # If you would like to clear your cache, go ahead and do a
 # "zsh-pip-clear-cache".
 
+# Adding pip3 alias if pip is missing
+if (( $+commands[pip3] && !$+commands[pip] )); then
+  alias pip=pip3
```

Comment:
This does not work because pip is realised in line 90: https://github.com/ohmyzsh/ohmyzsh/pull/10431/files#diff-c1da1ead97d68fd7f4967aacbf6df7b3b2406d0dd033b810fe93445c6d1539f6R90

Reply:
now it's done @mcornella